js/deer-config.js

```diff
@@ -58,5 +61,5 @@ export default {
         cat: (obj) => `<h5>${obj.name}</h5><img src="http://placekitten.com/300/150" style="width:100%;">`
```

Comment:
This is also in deer.js, and that is the one that gets used.

Reply:
yup. Just a placeholder to show how it works.